docs/contributing/code/building.md

```diff
@@ -91,3 +91,10 @@ The `fast` profile helps to significantly reduce build time, which is useful for
 build/mvn clean package -Pfast
 ```
 
+## Build with Web UI
+
+The Kyuubi Release 1.7.1 and earlier does not include the Web UI module, you should build Kyuubi Binary from master branch.
```

Comment:
Which version does this feature get involved in? 1.7.1 or 1.8.0

Reply:
1.8.0